diem-unified-charter.md

```diff
@@ -21,29 +21,40 @@ drivers in the development of a digital emblem architecture.
 # Initial Scope
 
 The working group will focus its initial work on assets that are discovered by use
-of digital communication as the primary interaction method that have the
+of digital communication such as use of DNS or at a well-know location on a host identified
+by a hostname or IP address. These discovery mechansisms have the
```

Comment:
I removed "primary interaction" based on a comment from Roman and added DNS and hostname/IP address as example based on Orie's text. However, I don't think it needs to be restricted to these in the charter.

Reply:
Note that this already talks about discovery by saying "discovered by use of digital communication" therefore I think "primary interaction" was redundant. Or am I missing something?